custom_components/frigate/icons.py

```diff
@@ -0,0 +1,80 @@
+"""Handles icons for different entity types."""
+
+from xmlrpc.client import Boolean
+
+ICON_BICYCLE = "mdi:bicycle"
+ICON_CAR = "mdi:car"
+ICON_CAT = "mdi:cat"
+ICON_CONTRAST = "mdi:contrast-circle"
+ICON_CORAL = "mdi:scoreboard-outline"
+ICON_COW = "mdi:cow"
+ICON_DOG = "mdi:dog-side"
+ICON_FILM_MULTIPLE = "mdi:filmstrip-box-multiple"
+ICON_HORSE = "mdi:horse"
+ICON_IMAGE_MULTIPLE = "mdi:image-multiple"
+ICON_MOTION_SENSOR = "mdi:motion-sensor"
+ICON_MOTORCYCLE = "mdi:motorbike"
+ICON_OTHER = "mdi:shield-alert"
+ICON_PERSON = "mdi:human"
+ICON_SERVER = "mdi:server"
+ICON_SPEEDOMETER = "mdi:speedometer"
+
+ICON_DEFAULT_ON = "mdi:home"
+
+ICON_CAR_OFF = "mdi:car-off"
+ICON_DEFAULT_OFF = "mdi:home-outline"
+ICON_DOG_OFF = "mdi:dog-side-off"
+
+
+def get_dynamic_icon_from_type(obj_type: str, is_on: Boolean) -> str:
+    """Get icon for a specific object type and current state."""
+
+    if obj_type == "car":
+        if is_on:
+            return ICON_CAR
+
+        return ICON_CAR_OFF
+    if obj_type == "dog":
+        if is_on:
+            return ICON_DOG
+
+        return ICON_DOG_OFF
+    if is_on:
+        return ICON_DEFAULT_ON
+
+    return ICON_DEFAULT_OFF
```

Comment:
```suggestion
        return ICON_DEFAULT_ON if is_on else ICON_DEFAULT_OFF
```

Reply:
I hated the way I did it but wasn't sure how to clean it up, not sure why I didn't think of that but that is much better 👌